ecosystem/typescript/sdk_v2/src/utils/hexData.ts

```diff
@@ -0,0 +1,67 @@
+import { bytesToHex, hexToBytes } from "@noble/hashes/utils";
+import { Hex } from "../types";
+
+export class HexData {
```

Comment:
My initial understanding was that this was going to just be a class full of static methods, not something you can instantiate. If we're going with this approach, it should probably be in types, not utils. Also it's weird that this is something you can instantiate but then it's just all static methods mostly.